src/game/main.cpp

```diff
@@ -269,6 +283,49 @@ void Create_Window()
 #endif
 }
 
+namespace Thyme
+{
+bool Create_Window_SDL2()
+{
+#ifdef BUILD_WITH_SDL2
+    g_applicationWindow = NULL;
+
+    if (SDL_Init(SDL_INIT_VIDEO) < 0) {
+        captainslog_error("SDL could not initialize!");
+        return false;
+    }
+
+    g_creatingWindow = true;
+
+    bool is_windowed = g_gameIsWindowed;
+    g_applicationWindow = SDL_CreateWindow(WIN_TITLE,
```

Comment:
According to documentation this call can return null. This seems to be not handled here.
https://wiki.libsdl.org/SDL_CreateWindow